docs/source/advanced/transfer_learning.md

```diff
@@ -0,0 +1,28 @@
+## Transfer Learning with ml4ir
+
+ml4ir saves individual layer weights as part of the `RelevanceModel.save(...)` call. These layer weights can be used with other ml4ir models for transfer learning. This enables layers like embedding vectors to be shared across search tasks like ranking, classification, etc. with ease.
+
+![serving](/_static/ml4ir_savedmodel.png)
```

Comment:
[minor] The alt text is displayed on the right of the image wit safari
This is weird

![image](https://user-images.githubusercontent.com/63343516/95357050-ab46f380-08c7-11eb-9cf0-6d8f5ca1a313.png)

Reply:
I noticed this on chrome too for some images. I think it is a bug of using markdown in sphinx. I will try to fix this. Can't find any help online.